OracleGoldenGate/21c/README.md

```diff
@@ -143,13 +158,15 @@ Oracle GoldenGate Administration Client for Oracle
 Version 21.3.0.0.0 ...
 ```
 
-### Running scripts before setup and on startup
-The container images can be configured to run scripts before setup and on startup. Currently, `.sh` extensions are supported. For setup scripts just mount the volume `/u01/ogg/scripts/setup` or extend the image to include scripts in this directory. For startup scripts just mount the volume `/u01/ogg/scripts/startup` or extend the image to include scripts in this directory. Both of those locations are static and the content is controlled by the volume mount.
+### Running Scripts Before Setup and on Startup
+
+The container images can be configured to run scripts before setup and on startup. Currently, `.sh` extensions are supported. For setup scripts just mount the volume `/u01/ogg/scripts/setup` or extend the image to include scripts in this directory. For startup scripts just mount the volume `/u01/ogg/scripts/startup` or extend the image to include scripts in this directory. Both of those locations
+are static and the content is controlled by the volume mount.
 
 The example below mounts the local directory myScripts to `/u01/ogg/scripts` which is then searched for custom startup scripts:
```

Comment:
myScripts needs to be wrapped in back ticks and is missing a leading forward slash.